packages/api-headless-cms/package.json

```diff
@@ -39,9 +39,11 @@
     "@webiny/utils": "^5.23.1",
     "@webiny/validation": "^5.23.1",
     "boolean": "^3.0.2",
+    "code-frame": "^5.0.0",
```

Comment:
Is it worth using @babel/code-frame as there are type definitions for it and it has had more recent updates?

Reply:
@endymion1818 Bundle size difference is significant for something that will rarely kick in: 14Kb vs 700B. Honestly I'm not even sure we need this at all, but let's have that in the error response for now. TS types are irrelevant here, because it's a single function call. The function is so small that we could even have our own utility for it.

https://bundlephobia.com/package/@babel/code-frame@7.16.7
vs
https://bundlephobia.com/package/code-frame@5.0.0

Reply:
@endymion1818 I reviewed this once more, and completely removed `code-frame`. Instead, I'm returning the full SDL, without any formatting. That way if something goes wrong, the developer can share the full SDL with us.